src/bots/modchat/messageHandler.js

```diff
@@ -0,0 +1,51 @@
+import { DMChannel } from 'discord.js';
+import client from '../../client';
+import generateEmbed from '../../utils/embed';
+import { getStrings } from './constants';
+import { v4 as uuidv4 } from 'uuid';
+import { getModChannel, parseCommandString } from '../../utils/index';
+
+/**
+ * Handles direct messages from users to forward to moderation team.
+ */
+let handlePrivateMessage = async () => {
+  let cmd = parseCommandString();
+  let channel = client.message.channel;
+  let strings = getStrings();
+
+  if (cmd.arguments.length === 0) {
+    channel.send(strings.explanation);
+    return;
+  }
+
+  if (!(channel instanceof DMChannel)) {
+    channel.send(strings.dmOnly);
+    return;
+  }
+
+  let footer = uuidv4();
+
+  let modChannel = await getModChannel();
+
+  let title =
+    cmd.subCommand === '-a'
+      ? 'Anonymous'
+      : `${client.message.author.username}#${client.message.author.discriminator}`;
+
+  let description = cmd.arguments.join(' ');
+
+  if (cmd.subCommand !== '-a') {
+    description = `${cmd.subCommand} ${description}`;
+  }
+
+  let embed = generateEmbed({
+    description,
+    footer,
+    title
+  });
+
+  await modChannel.send(embed);
+  channel.send(getStrings([footer]).success);
```

Comment:
i don't feel like this is a really good pattern. ideally we'd just put in a string that would act as a key and return the value... maybe change options to a map?

Reply:
It matches the other bots. This one I can change to array, but otherwise I'd want to keep consistent

Reply:
oh i see what's happening... can we not call the ID footer? that's confusing